general-concepts/licenses/text.xml

```diff
@@ -6,8 +6,28 @@
 <body>
 <p>
 All ebuilds must specify a <c>LICENSE</c> (note the American English
-spelling) variable. The value of this variable must be the name of a
-file in the Gentoo repository's <c>licenses/</c> directory.
+spelling) variable. The license names listed in this variable must
+match files existing in the repository's <c>licenses/</c> directory.
+</p>
+
+<p>
+The value of this variable should be the union of licenses pertaining
+to all files installed by the package. If some parts of the package
```

Comment:
"Union" might imply a logical OR. So maybe better "... should include all licenses pertaining to the files installed by the package."